README.md

```diff
@@ -12,6 +12,7 @@ This repository includes optimized deep learning models and a set of demos to ex
 * [Model Downloader](tools/downloader/README.md) and other automation tools
 * [Demos](demos/README.md) that demonstrate models usage with Deep Learning Deployment Toolkit
 * [Accuracy Checker](tools/accuracy_checker/README.md) tool for models accuracy validation
+* [Model Contribution Guide](contribution.md)
```

Comment:
I think the right place for link in "How to contribute" section

Reply:
I thought that way too. But I decided, that it would be better to make it as notable as possible. I can add link to `How to contribute` section too.

Reply:
I would say this link ought to _replace_ the "How to Contribute" section.